entries/accordion.xml

```diff
@@ -86,6 +86,21 @@
 				</desc>
 			</type>
 		</option>
+		<option name="classes" type="Object" default='Object' example-value='{ "ui-accordion-header": null }'>
```

Comment:
Yeah, but we can just create custom examples that use `"classes.ui-accordion-header"` instead of `"classes"`.